SwiftRNBO_Example_multiplatfrom_SwiftUI/SwiftRNBO_Example_multiplatfrom_SwiftUI/RNBO/Interface/RNBOAudioEngine.swift

```diff
@@ -16,8 +16,8 @@ class RNBOAudioEngine {
     
     private func initInput() {
         let input = self.engine.inputNode
-        //let format = input.inputFormat(forBus: 0)
-        let format = avAudioUnit!.inputFormat(forBus: 0)
+        let format = input.inputFormat(forBus: 0)
```

Comment:
We had to uncomment this in order a crash not to happen on some systems. The error was about channel count mismatch.